torchci/components/WorkflowBox.tsx

```diff
@@ -21,24 +22,23 @@ function sortJobsByConclusion( jobA: JobData, jobB: JobData): number {
 function getWorkflowJobSummary(job: JobData) {
   var queueTimeInfo = null
   if (job.queueTimeS != null) {
-    queueTimeInfo = <><i>Queued:</i> {Math.max(Math.round(job.queueTimeS / 60), 0)} mins</>
+    queueTimeInfo = <><i>Queued:</i> {Math.max(Math.round(job.queueTimeS / 60), 0)} mins</>;
   }
 
   var durationInfo = null
   if (job.durationS != null) {
-    durationInfo = <><i>Duration:</i> {Math.round((job.durationS / 60))} mins</>
+    durationInfo = <><i>Duration:</i> {Math.round((job.durationS / 60))} mins</>;
   }
 
   var separator = (queueTimeInfo && durationInfo) ? ", ": ""
 
   return <>
     <JobSummary job={job} />
-    <br/>
     <small>
-      &nbsp;&nbsp;&nbsp;&nbsp;
-      {queueTimeInfo}{separator}{durationInfo}
+      |&nbsp;
+      {queueTimeInfo}{separator}{durationInfo}<TestInsightsLink job={job} separator={", "} />
     </small>
-  </>
+  </>;
```

Comment:
(weakly held opinion)
This is subjective, but I thought it looked neater & more skimmable on two separate lines :) 
Plus, some job names are so long it wraps around to the next line anyways

Reply:
I see. I have no strong opinion here, only remember Suo thoughts a while ago about how long the page is with queue time and stuffs come right after every job, so I thought I could try to make it one liner. Let me put this back as before unless there is strong objection for that :)